dbms/sqlquery/mysql/src/cloud/benchflow/monitors/querymysql.go

```diff
@@ -39,19 +40,30 @@ func queryHandler(w http.ResponseWriter, r *http.Request) {
         }
         if name == value {
         	if(method == "equal") {
-        		fmt.Fprintf(w, "Row "+strconv.Itoa(rowI)+" matches "+value+" \n")
+        		//fmt.Fprintf(w, "Row "+strconv.Itoa(rowI)+" matches "+value+" \n")
+        		response = Response{true}
         		}
         	}
         if name != value {
         	if(method == "nequal") {
-        		fmt.Fprintf(w, "Row "+strconv.Itoa(rowI)+" doesn't match "+value+" \n")
+        		//fmt.Fprintf(w, "Row "+strconv.Itoa(rowI)+" doesn't match "+value+" \n")
+        		response = Response{true}
         		}
         	}
     }
     db.Close()
+    
+    js, err := json.Marshal(response)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+	    return
+    }
+    
```

Comment:
Fine

Vincenzo Ferme

Skype: vincenzoferme
Web: http://www.vincenzoferme.it
Email: info@vincenzoferme.it
On May 31, 2016 10:05 AM, "Gabriele Cerfoglio" <notifications@github.com>
wrote:

> In dbms/sqlquery/mysql/src/cloud/benchflow/monitors/querymysql.go
> <https://github.com/benchflow/monitors/pull/19#discussion_r65137974>:
>
> >          		}
> >          	}
> >      }
> >      db.Close()
> > +
> > +    js, err := json.Marshal(response)
> > +	if err != nil {
> > +		http.Error(w, err.Error(), http.StatusInternalServerError)
> > +	    return
> > +    }
> > +
>
> @VincenzoFerme <https://github.com/VincenzoFerme> It's set by default on
> 200 if you send data, even empty. Other parts it's specified.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/benchflow/monitors/pull/19/files/1d99b54cec5c903516fc46c26c5dda469b79ffb4..88288c8c63362c49de6ee5089dc8ab418570c384#r65137974>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG1O-cIOnuyiaAqff7vbMoc7gqbq0kgkks5qG-vegaJpZM4ImxyI>
> .
>